examples/platforms/cc2652/README.md

```diff
@@ -0,0 +1,115 @@
+# OpenThread on CC2652 Example
+
+This directory contains example platform drivers for the [Texas
+Instruments CC2652R1][cc2652r1].
+
+The example platform drivers are intended to present the minimal code necessary
+to support OpenThread.  As a result, the example platform drivers do not
+necessarily highlight the platform's full capabilities.  The platform drivers
+were build for the [CC2652R1 LAUNCHXL][cc2652r1-launchxl], usage on other
+boards with a cc2652r1 may require changes to the peripheral drivers.
+
+<!---
+TODO: update documentation with live links
```

Comment:
The LaunchPad link is live and I need to update it. The CC2652R1 product page is not live and won't be until we release the product to market, I'll move the TODO to reflect that this link needs to be updated when the product page is live.